### ISSUE TITLE
Normals are "a bit funky"

### ISSUE THREAD

**Issue by robosoup2** [thnewlands/3D-Texture-Brush-for-Mudbun#4]
I baked a very high poly sphere (with a little edge room for blending) to compare with a normal mudsphere. the normals look a little wacky especially when you transform the brush around.
obviously the SDF is limited by its 3DTex resolution, but it's pretty obvious even with mudbun voxels set to MUCH lower resolution than the 3DTex itself 🤔 maybe there is a different way values could be sampled? but I have no idea what i'm talking about lol

I recorded video of it at some different voxel resolutions. but the compression makes it hard to see 😅 it's very obvious in-person though

https://streamable.com/ignz84

![image](https://github.com/thnewlands/3D-Texture-Brush-for-Mudbun/assets/50124081/e01cace9-13e0-49d1-84ec-4ff3f264e43e)

**Comment by thnewlands** [comment]
Yeah! One of the things I was looking into was adding a smoothing variable by interpolating between LODs. It started introducing artifacts as other samples started to bleed in. 

Another approach to this would be making the extension configurable so that you could have 256x256x256 textures instead of 128 which might help with aliasing.

**Comment by robosoup2** [comment]
interesting! yeah tbh i'm not sure how much the SDF resolution would make a difference, bc setting mudbun to really low poly (even lower than in my vid) still shows artifacts 🤔 but maybe I can put some normal maps on it and nobody will notice haha

**Comment by robosoup2** [comment]
okay good news, looks like upping the Smooth Normal Blur setting on a mud renderer to just 0.01 is enough to clean up the artifacts pretty well without heavily changing the appearance of other primitives. you can still see it a bit on the sphere (especially if you rotate it) but I think that's enough to make it usable!

(before and after; blue ones are 3DTex primitives)
![image](https://github.com/thnewlands/3D-Texture-Brush-for-Mudbun/assets/50124081/23cab5a3-3a60-4321-8523-751286a770e3)
![image](https://github.com/thnewlands/3D-Texture-Brush-for-Mudbun/assets/50124081/15ab8c2d-deda-469e-bfdb-9cba86f012b0)

**Comment by thnewlands** [comment]
Awesome!! That's a great fix.
